blog/2020-11-09-atomic-swaps-on-liquid.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Confidential Atomic Swaps on Liquid"
+author: philipp
+author_url: https://github.com/bonomat
+author_image_url: https://avatars2.githubusercontent.com/u/224613
+tags: [dev-update]
```

Comment:
🤔 I always forget the tags!

Reply:
Probably doesn't matter much, but thought to add it to `dev-update`